CHANGES.rst

```diff
@@ -278,6 +278,11 @@ astropy.extern
 astropy.io.ascii
 ^^^^^^^^^^^^^^^^
 
+  - Options not available with the fast reader raise a ``ParameterError``
```

Comment:
Overall this needs to be moved to 3.1.

The API changes (in particular the two changes to exception class) need to go in the API changes section.  Changes that are fixing an actual bug (like a read option being ignored) should go in Bug fixes.  Then changes that improve performance (like using a fast reader instead of a slow reader, adding RST to the guess list) go into New Features.